Misc/NEWS.d/next/Build/2019-12-16-17-50-42.bpo-31046.XA-Qfr.rst

```diff
@@ -0,0 +1 @@
+A directory prefix can now be specified when using :mod:`ensurepip`.
```

Comment:
Can you move this into `Library` directory, rather than being in `Build`?

Reply:
Moved in 462211527b0d65233a86652d19c9b6e192e5a13b